src/Integration.UnitTests/Binding/CSharpVBBindingConfigProviderTests.cs

```diff
@@ -48,14 +48,27 @@ public class CSharpVBBindingConfigProviderTests
         private IList<NuGetPackageInfo> validNugetPackages;
         private Core.CSharpVB.RuleSet validRuleSet;
 
+        private static readonly SonarQubeRule ActiveRuleWithUnsupportedSeverity = new SonarQubeRule("activeHotspot", "any1",
+            true, SonarQubeIssueSeverity.Blocker, null, SonarQubeIssueType.SecurityHotspot);
+
+        private static readonly SonarQubeRule InactiveRuleWithUnsupportedSeverity = new SonarQubeRule("inactiveHotspot", "any2",
+            false, SonarQubeIssueSeverity.Blocker, null, SonarQubeIssueType.SecurityHotspot);
+
+        private static readonly SonarQubeRule ActiveTaintAnalysisRule = new SonarQubeRule("activeTaint", "roslyn.sonaranalyzer.security.foo",
+            true, SonarQubeIssueSeverity.Blocker, null, SonarQubeIssueType.CodeSmell);
+
+        private static readonly SonarQubeRule InactiveTaintAnalysisRule = new SonarQubeRule("inactiveTaint", "roslyn.sonaranalyzer.security.bar",
+            false, SonarQubeIssueSeverity.Blocker, null, SonarQubeIssueType.CodeSmell);
+
+
         [TestInitialize]
         public void TestInitialize()
         {
             emptyRules = Array.Empty<SonarQubeRule>();
 
             validRules = new List<SonarQubeRule>
             {
-                new SonarQubeRule("key", "repoKey", true, SonarQubeIssueSeverity.Blocker, null, SonarQubeIssueType.Unknown)
```

Comment:
The filtering by issue type now means that `Unknown` means the rule isn't supported.